src/chrome/content/rules/Pando.xml

```diff
@@ -7,7 +7,7 @@
 		- pandoblog.com                (dm1.fastdomain.com)
 		- www.pandoblog.com            (dm1.fastdomain.com)
 -->
-<ruleset name="Pando (bad certs)" default_off="Wrong hostnames in certs.">
```

Comment:
jsha, 21/01/2015 20:37:
> Just suggesting that the test can help find some of the issues.

Yep. I didn't know about the test script, I'll use it too